### PR TITLE
Feature/multipe pages

### PR DESCRIPTION
**Pull request by sanderytt**
Added feature of adding multiple pages (now only with the click of a button)

### CODE REVIEW

File: src/pages/page-2.js [hunk 1]
```diff
@@ -1,151 +1,127 @@
...
-    const newArr = [...this.state.workexp];
+    //const newArr = [...this.state.workexp];
+    const newArr = [...isWorkExp]
 
     //Change the targeted item
     newArr[index][targetedValue] = newValue;
 
     //Setting the state with changed array
-    this.setState({workexp: newArr});
+    //this.setState({workexp: newArr});
```
Comment (Giggleball) [inline src/pages/page-2.js]: Can this be removed?

File: src/pages/page-2.js [hunk 2]
```diff
@@ -1,151 +1,127 @@
...
-
-    this.setState({headerinfo: newObj})
-  }
-
-  changeWorkExp(e, index) {
-    //Get values of event
-    const targetedValue = e.target.id;
-    const newValue = e.target.value;
-    const newArr = [...this.state.workexp];
+    //const newArr = [...this.state.workexp];
```
Comment (Giggleball) [inline src/pages/page-2.js]: Remove your commented out code if you won't use it anymore.

Comment (sanderytt) [reply src/pages/page-2.js]: Can't find this comment in the code, I think I removed it

File: src/pages/page-2.js (hunk 1 above)
Comment (sanderytt) [reply src/pages/page-2.js]: Can't find this comment in the code, I think I removed it

**Pull request merged**